virttest/utils_test/qemu/__init__.py

```diff
@@ -498,13 +499,30 @@ def get_guest_free_mem(cls, vm):
         :return: free memory report by guest OS in MB
         """
         os_type = vm.params.get("os_type")
-        timeout = float(vm.params.get("login_timeout", 600))
+        timeout = vm.params.get_numeric("login_timeout", 600, float)
         try:
             session = vm.wait_for_login(timeout=timeout)
             return utils_misc.get_free_mem(session, os_type)
         finally:
             session.close()
 
+    @classmethod
+    def _get_guest_crashkernel_size(cls, vm):
```

Comment:
```suggestion
    def _get_linux_guest_crashkernel_size(cls, vm):
```
Currently, we just support this workaround in the Linux guest, so let's add the `linux_` prefix.

Reply:
Corrected